cmd/lukso/validator.go

```diff
@@ -48,174 +47,223 @@ type DepositDataKey struct {
 	DepositCliVersion     string   `json:"deposit_cli_version"`
 }
 
-func sendDeposit(ctx *cli.Context) error {
-	// Dialing clients and creating bindings
+type depositController struct {
+	c context.Context
+
+	eth        *ethclient.Client
+	genDep     *bindings.LYXe
```

Comment:
please use full names, like `genesisDeposit` and `deposit` there

Reply:
Done